src/MotoROS2_AllControllers.vcxproj.filters

```diff
@@ -285,6 +300,9 @@
     <ClCompile Include="ServiceStartPointQueueMode.c">
       <Filter>Source Files\Services</Filter>
     </ClCompile>
+    <ClCompile Include="Ros_mpGetRobotCalibrationData.c">
+      <Filter>Source Files\Robot Controller</Filter>
```

Comment:
@ted-miller: feel free to move this to another 'location'.

Reply:
This is fine where it is